share-coordinator/src/main/java/org/apache/kafka/coordinator/share/ShareCoordinatorShard.java

```diff
@@ -317,16 +319,12 @@ public CoordinatorResult<WriteShareGroupStateResponseData, CoordinatorRecord> wr
 
         CoordinatorRecord record = generateShareStateRecord(partitionData, key);
         // build successful response if record is correctly created
-        WriteShareGroupStateResponseData responseData = new WriteShareGroupStateResponseData()
-            .setResults(
-                Collections.singletonList(
-                    WriteShareGroupStateResponse.toResponseWriteStateResult(key.topicId(),
-                        Collections.singletonList(
-                            WriteShareGroupStateResponse.toResponsePartitionResult(
-                                key.partition()
-                            ))
-                    ))
-            );
+        WriteShareGroupStateResponseData responseData = new WriteShareGroupStateResponseData().setResults(
```

Comment:
I think the previous style of

```
new WriteShareGroupStateResponse()
.setResults(
```

was more in line with the rest of the code you've written.

Reply:
Will need to refactor all the code - this is the acceptable style since previous one causes alignment issues.
I have a JIRA for the same

Reply:
It will need to be changed to as older style causes bad alignment. I have a JIRA for the same.